jenkins/pipelines/cd/tiup/tiup-mirror-update-multi-products.groovy

```diff
@@ -645,14 +616,13 @@ node("build_go1130") {
                                 // if (RELEASE_TAG == "nightly" || RELEASE_TAG >= "v5.3.0") {
                                 //     update "dm", RELEASE_TAG, dm_sha1, "darwin", "amd64"
                                 // }
-                                // update_ctl RELEASE_TAG, "darwin", "arm64"
+                                // using self built etcdctl
```

Comment:
Tiny: any issue reference?

Reply:
no issue, the original author even do not remember why.
I figure it out during this work